### ISSUE TITLE
Accessibility: output terminals inaccessible.

### ISSUE THREAD

**Issue by derekriemer** [Microsoft/vscode#36150]
- VSCode Version: Code - Insiders 1.18.0-insider (c098b394b07afbb58975ab609117a0e115c5a3fa, 2017-10-12T05:24:34.433Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
jshint|dba|0.10.15
python|don|0.7.0
cpptools|ms-|0.13.1
incremental-search|sie|0.2.1
code-spell-checker|str|1.4.8
JavaScriptSnippets|xab|1.5.0

---

Steps to Reproduce:

1. Create a new node.js project.
2. place this in .babelrc
    ```
    {
      "presets": ["env"]
    }
    ```
3. in package.jsonconfigure this.
This is a sample package.json.
    ```
    {
      "name": "elizabot",
      "version": "1.0.0",
      "description": "test",
      "main": "lib/index.js",
      "dependencies": {
        "babel-cli": "^6.26.0",
        "jshint": "^2.9.5"
      },
      "devDependencies": {
        "babel-preset-env": "^1.6.0"
      },
      "scripts": {
        "build": "babel src -d lib",
        "test": "echo \"Error: no test specified\" && exit 1"
      },
      "author": "",
      "license": "ISC"
    }
    ```
5. Now, type this in src/index.js
    ```
    you will fail to transpile!
    ```
4. Build. The program asks to configure default build task. If you use the node task, it'll work. My tasks.json is like this however.
    ```
    {
        // See https://go.microsoft.com/fwlink/?LinkId=733558
        // for the documentation about the tasks.json format
        "version": "2.0.0",
        "tasks": [
            {
                "type": "npm",
                "script": "build",
                "problemMatcher": [
                    "$eslint-compact"
                ]
            }
        ]
    }
    ```

Expected: With the NVDA screen reader running, vscode's output window is readable.

Actual: There's no text whatsoever visible with NVDA. However, if I OCR the whole window, text is visible in the console.
I've reproduced this with a python project with custom tasks.json as well.

Reproduces without extensions: Yes

**Comment by derekriemer** [comment]
This issue is a show stopper for blind developers who want to use tasks, and at least in previous builds of vscode, the tasks pannel used to be accessible.

**Comment by Tyriar** [comment]
Duplicate https://github.com/Microsoft/vscode/issues/8339 - we'll be working on this soon

**Comment by derekriemer** [comment]
Okay, thanks. I'll go subscribe there.
